src/com/dabomstew/pkrandom/newgui/NewRandomizerGUI.java

```diff
@@ -25,6 +25,7 @@
 /*--  along with this program. If not, see <http://www.gnu.org/licenses/>.  --*/
 /*----------------------------------------------------------------------------*/
 
+import cli.CliRandomizer;
```

Comment:
Need to change this import since the package was changed.
```suggestion
import com.dabomstew.pkrandom.cli.CliRandomizer;
```